charts/geti-tools/charts/mongodb/values.yaml

```diff
@@ -13,7 +13,7 @@ auth:
 resources:
   requests:
     cpu: 400m
-    memory: 2Gi
+    memory: 1Gi
```

Comment:
Did you run any experiment to measure the impact of these changes on performance? Several of the request/limits were tuned last year by @AlbertvanHouten (PR #11443 in the old repo) to optimize performance & autoscaling behavior.

Reply:
Reducing the requests should be fine in most cases if the system has spare memory to grow to the limit.

Reply:
I tested it several times - with various models - and I didn't notice any special impact on performance.